src/DataTable/TableRow.js

```diff
@@ -56,75 +56,80 @@ const TableRowStyle = styled.div`
   ${props => props.pointerOnHover && pointerCSS};
 `;
 
-const TableRow = memo(({
-  id,
-  keyField,
-  columns,
-  row,
-  onRowClicked,
-  selectableRows,
-  expandableRows,
-  striped,
-  highlightOnHover,
-  pointerOnHover,
-  expandableRowsComponent,
-  expandableDisabledField,
-  defaultExpanded,
-}) => {
-  const [expanded, setExpanded] = useState(defaultExpanded);
-  const handleRowClick = useCallback(e => {
-    // use event delegation allow events to propogate only when the element with data-tag __react-data-table--click-clip___ is present
-    if (e.target && e.target.getAttribute('data-tag') === '___react-data-table--click-clip___') {
-      onRowClicked(row, e);
-    }
-  }, [onRowClicked, row]);
-  return (
-    <>
-      <TableRowStyle
-        id={`row-${id}`}
-        striped={striped}
-        highlightOnHover={highlightOnHover}
-        pointerOnHover={pointerOnHover}
-        onClick={handleRowClick}
-        className="rdt_TableRow"
-      >
-        {selectableRows && (
-          <TableCellCheckbox
-            name={`select-row-${row[keyField]}`}
-            row={row}
-          />
-        )}
+const TableRow = memo(
```

Comment:
Can you restore the original formatting here? Looks like you just changed ` selectableRowsDisabledField: PropTypes.string.isRequired,` so that should be the only diff.

Reply:
Actually, see this comment https://github.com/jbetancur/react-data-table-component/pull/188/files#r312010155

We can just pass this down using context and avoid prop drilling.